conf/config-schema.json

```diff
@@ -9,7 +9,8 @@
         "settings": { "type": "object" },
         "extends": { "type": ["string", "array"] },
         "rules": { "type": "object" },
-        "parserOptions": { "type": "object" }
+        "parserOptions": { "type": "object" },
+        "eslintIgnore": { "type": ["array"] }
```

Comment:
I think the `eslintIgnore` option shouldn't be part of the config -- it should be outside of the `eslintConfig` object. So a `package.json` file might look like this:

```js
{
  "eslintConfig": {
    "rules": {
      "semi": "error"
    }
  },
  "eslintIgnore": [
    "*.test.js"
  ]
}
```